packages/main/src/TimePicker.hbs

```diff
@@ -0,0 +1,70 @@
+<div id="{{this._id}}" formatPattern="{{formatPattern}}" class="sapMTimePicker">
+	<ui5-input
+			id="{{_id}}-inner"
+			value="{{value}}"
+			?disabled="{{disabled}}"
+			?readonly="{{readonly}}"
+			value-state="{{valueState}}"
+			@ui5-change="{{_handleInputChange}}"
+			@ui5-input="{{_handleInputLiveChange}}"
+	>
+		<ui5-icon
+			slot="icon"
+			name="time-entry-request"
+			tabindex="-1"
+			show-tooltip
+			@click="{{togglePicker}}"
+			input-icon
+			?pressed="{{_isPickerOpen}}"
+		></ui5-icon>
+	</ui5-input>
+	<ui5-popover
+			id="{{_id}}-popover"
+			allow-target-overlap="{{_popover.allowTargetOverlap}}"
+			placement-type="{{_popover.placementType}}"
+			no-arrow
+			horizontal-align="{{_popover.horizontalAlign}}"
+			stay-open-on-scroll="{{_popover.stayOpenOnScroll}}"
+			@ui5-afterClose="{{_popover.afterClose}}"
+			@ui5-afterOpen="{{_popover.afterOpen}}"
+			header-text="Pick Time"
+			class="sapMTimePickerPopover"
+		>
+		{{#if shouldBuildHoursSlider}}
+			<ui5-slider 
+				label = "Hours"
+				value="6"
```

Comment:
Imo initialize either with user input or with current time, not hardcoded values

Reply:
This was a test value, and will be removed in the end version. Most probably will be as you said.